kogito-cloud-operator.jenkinsfile

```diff
@@ -286,6 +288,25 @@ void installOpenshiftClient(){
     """
 }
 
+void installGraalVMWithNative(){
+    echo "installGraalVMWithNative"
+    sh """       
+        rm -rf ${WORKSPACE}/graalvm*
+        RELEASE_VERSION=19.3.1
```

Comment:
Wouldn't it have more sense to put to `environment` above so we don't have to go through whole script when we need to raise the version?

Reply:
right
will set it

Reply:
done